### PR TITLE
feat(tooling): terraform initial setup

### PR DESCRIPTION
**Pull request by m-juckes-pins**
# DO NOT MERGE

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: infrastructure/environments/dev/provider.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    # key                  = "dev.tfstate"
+  }
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.99.0"
+    }
+  }
+  required_version = ">= 1.1.6, < 3.99.0"
```
Comment (beejjacobs) [inline infrastructure/environments/dev/provider.tf:16]: If I follow the docs correctly, this should be the terraform version, currently we use 1.7.3.

For DaRT I've gone for: `>= 1.7.3, < 1.8`

File: appeals/web/src/client/components/file-uploader/_errors.js
Comment (m-juckes-pins) [inline appeals/web/src/client/components/file-uploader/_errors.js:1]: ignore - included in pr after squashing commits

File: appeals/web/src/client/components/file-uploader/_html.js
Comment (m-juckes-pins) [inline appeals/web/src/client/components/file-uploader/_html.js:1]: ignore - included in pr after squashing commits

**Comment by beejjacobs** [comment]
Superseded by other PRs.

**Pull request closed without merging**